lambdas/src/services/session-request-validator.ts

```diff
@@ -5,16 +5,16 @@ import { ClientConfigKey, ConfigKey } from "../types/config-keys";
 import { Logger } from "@aws-lambda-powertools/logger";
 import { SessionValidationError } from "../common/utils/errors";
 import { EvidenceRequest } from "./evidence_request";
-import { EvidenceRequestConfig } from "../types/evidence-requested-config";
+import { CRIEvidenceProperties } from "./cri_evidence_properties";
 export class SessionRequestValidator {
     constructor(
         private validationConfig: SessionRequestValidationConfig,
         private jwtVerifier: JwtVerifier,
-        private evidenceRequest?: EvidenceRequestConfig,
+        private criEvidenceProperties?: CRIEvidenceProperties,
     ) {}
     async validateJwt(jwt: Buffer, requestBodyClientId: string): Promise<JWTPayload> {
         const expectedRedirectUri = this.validationConfig.expectedJwtRedirectUri;
-        const configuredStrengthScore = this.evidenceRequest?.strengthScore;
+        const configuredStrengthScore = Number(this.criEvidenceProperties?.strengthScore);
```

Comment:
Why are we doing `Number()`? The type defines `strengthScore` as an optional number, so we know that it's a number if we have it and if we don't this will produce `NaN`?